naming/program/src/state.rs

```diff
@@ -0,0 +1,47 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use solana_program::{msg, program_error::ProgramError, pubkey::Pubkey};
+
+#[derive(Debug, PartialEq, BorshDeserialize, BorshSerialize, Clone, Copy)]
+pub enum NameStatus {
+    Deleted,
```

Comment:
What's `NameStatus::Deleted` for?  If the name is deleted then I'd think the account wouldn't exist.  Is this really `NameStatus::Uninitialized`?

Reply:
Ah yes it's more a `Unitialiazed` indeed, I'm going to fix it now.